src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -4001,6 +4002,9 @@ def groupby_apply(
                 Keyword arguments to pass to agg_func when applying it to each group.
             series_groupby:
                 Whether we are performing a SeriesGroupBy.apply() instead of a DataFrameGroupBy.apply()
+            include_groups:
+                When True, will apply func to the groupings in the case that
```

Comment:
```suggestion
                When True, will include grouping keys when calling func in the case that
```

Minor nit: I feel like the current documentation provided by pandas is a little unclear, and that this may be clearer? Feel free to ignore if you disagree!

Reply:
Agreed; done.